pkg/build/strategies/sti/sti.go

```diff
@@ -371,12 +369,13 @@ func (b *STI) Save(config *api.Config) (err error) {
 		image = config.Tag
 	}
 	outReader, outWriter := io.Pipe()
+	defer outReader.Close()
+	defer outWriter.Close()
```

Comment:
This matches the pattern of neighboring code:
https://github.com/openshift/source-to-image/pull/423/files#diff-77c4df2e21c6a783c0928af25a29ba8dR431